### PR TITLE
[14.0][MIG] sale_stock_available_to_promise_release_cutoff: Migration to 14.0

### PR DESCRIPTION
**Pull request by hiepnh97**
Notes:
This PR Depend on:
- <s>[PR 1403](https://github.com/OCA/sale-workflow/pull/1403)</s>
- https://github.com/OCA/sale-workflow/pull/1736

I will delete `test-requirement.txt` when depending on PR 1403 was Merge.

Ref. 3009

### CODE REVIEW

**Comment by mmequignon** [comment]
This PR should depend on https://github.com/OCA/sale-workflow/pull/1736 instead.
Thanks !

**Comment by hailangvn** [comment]
Thanks @mmequignon for your comment. We will work on this.

**Comment by dzungtran89** [comment]
Hi @mmequignon , the PR has been updated. btw the travis failed because the dependency is missing in `setup` dir`

**Comment by sebalix** [comment]
Thank you @hiepnh97 and @dzungtran89

**Review by mmequignon — APPROVED** [review]

File: sale_stock_available_to_promise_release_cutoff/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Glue module between `sale_stock_available_to_promise_release` and `sale_cutoff_time_delivery`.
```
Comment (TDu) [inline sale_stock_available_to_promise_release_cutoff/readme/DESCRIPTION.rst]:
```suggestion
Glue module between `sale_stock_available_to_promise_release` and `sale_delivery_date`.
```

**Comment by TDu** [comment]
I restarted Travis, because I just added the missing setup directory there https://github.com/OCA/sale-workflow/pull/1736/commits

**Comment by dzungtran89** [comment]
@TDu thx for your update

**Review by TDu — APPROVED** [review]
LG, should be squashed when `sale_delivery_date` has been merged

**Comment by sebalix** [comment]
@dzungtran89 Hello! Commit `Update sale_stock_available_to_promise_release_cutoff/readme/DESCRIPTION.rst` can be squashed in the MIG commit I guess, and the TMP commit can be removed as `sale_delivery_date` has been merged

**Comment by hailangvn** [comment]
Thanks @sebalix for the comment, this PR has been updated.

**Comment by sebalix** [comment]
@hailangvn thank you!
/ocabot merge nobump

**Pull request merged**